src/Uno.Sdk/targets/Uno.Implicit.Packages.ProjectSystem.Desktop.targets

```diff
@@ -6,7 +6,7 @@
 	-->
 	<Import Project="$(MSBuildThisFileDirectory)Uno.Implicit.Packages.ProjectSystem.Uno.targets" />
 
-	<ItemGroup Condition="$(UnoFeatures.Contains(';canvas3dgl;'))">
+	<ItemGroup Condition="$(UnoFeatures.Contains(';GLCanvas;'))">
```

Comment:
This should be `glcanvas` (lowercase) I think we we normalize UnoFeatures to lower case